c++/src/kj/iovmax.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2013-2014 Sandstorm Development Group, Inc. and contributors
```

Comment:
Hm does seem like an appropriate place, but notice iovmax.h is only used internally within kj & doesn't form part of any public API. Are you okay with exposing what I'm effectively treating as an internal implementation detail?

In particular, I'm not providing an implementation of iovMax for Windows here because it the IOV_MAX thing doesn't really apply there. Should there be some stub implementation of iovMax for Windows just in case somebody tries to call iovMax on POSIX only to see it fail when they build under say MinGW?